security-analysis/security-analysis-api/pom.xml

```diff
@@ -59,6 +59,11 @@
             <artifactId>powsybl-iidm-api</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>powsybl-iidm-impl</artifactId>
+            <version>${project.version}</version>
+        </dependency>
```

Comment:
This dependency **should not** be added without the test scope here.
Otherwise, security analysis won't work anymore with custom IIDM implementations, such as the `powsybl-network-store` one.

Reply:
Reverting the scope change will lead to errors in `AbstractReducedLoadingLimits`.
I think that for this class and for its `ReducedTemporaryLimit` subclass, it is acceptable to not support properties. So I suggest to apply for them the same strategy as for `OverloadImpl.UNACCEPTABLE_LIMIT` (return null, empty set or false, or throw an UsupportedOperationException).